Code/GraphMol/DistGeomHelpers/Wrap/rdDistGeom.cpp

```diff
@@ -20,6 +20,8 @@
 #include <GraphMol/DistGeomHelpers/BoundsMatrixBuilder.h>
 #include <GraphMol/DistGeomHelpers/Embedder.h>
 
+// #include <GraphMol/ForceFieldHelpers/CrystalFF/TorsionPreferences.h>
```

Comment:
since you aren't using this you might as well remove it

Reply:
done